FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -520,19 +520,27 @@ extension ComposeViewController: ASTableDelegate, ASTableDataSource {
         }
     }
 
-    func tableNode(_: ASTableNode, didSelectRowAt indexPath: IndexPath) {
-        guard case let .searchEmails(emails) = state else { return }
-
-        switch indexPath.section {
-        case 1:
-            let selectedEmail = emails[safe: indexPath.row]
-            handleEndEditingAction(with: selectedEmail)
-        case 2:
-            askForContactsPermission()
-        default:
-            break
+    func tableNode(_ tableNode: ASTableNode, didSelectRowAt indexPath: IndexPath) {
+        if case let .searchEmails(emails) = state,
+           indexPath.section == 1 {
```

Comment:
Why we need `indexPath.section == 1` condition here?
In this case line 531 `askForContactsPermission()` won't be executed

Reply:
I don't know why we have this condition. Let's revert